src/tasks/familiarweight.ts

```diff
@@ -145,6 +151,10 @@ export const FamiliarWeightQuest: Quest = {
             tryAcquiringEffect($effect`Offhand Remarkable`);
           }
         }
+        if ($familiars`Comma Chameleon, Homemade Robot`.every((fam) => have(fam)) &&
+            commaWeight < 111) {
+          print(`Comma Chameleon is not at max weight, use ${10 - get("homemadeRobotUpgrades")} more parts on Homemade Robot.`, "red");
```

Comment:
Seems like the printout should be `9 - get("homemadeRobotUpgrades")` since the max we can get is 9. 

Also, I'm not understanding the `111` value here, but that aside, this warning probably should go after the `visitUrl("charpane.php")`. Specifically, we should be able to do something like
```
// (We do not need to worry about what famequip we are using, and it only triggers when we are in fact using the comma)
if (familiarWeight(myFamiliar()) < 106) // 9 * 11 + 6
  print("Warning");
```

Reply:
Moving it to after the charpane visit would mean the warning is not printed when comma is not being used for being underweight. I could move it up to where the weight is checked, so this all happens in one place.

Reply:
The numbers are off, have changed the 10 to a 9. Testing to see what the max comma weight should be, its probably either 100 or 105.